src/BinSkim.Sdk/SdkResources.resx

```diff
@@ -243,4 +243,7 @@
   <data name="MetadataCondition_ImageIsArm64BitBinary" xml:space="preserve">
     <value>image is an ARM64 binary</value>
   </data>
+  <data name="MetadataCondition_ImageBuiltWithoutRecordCommandLine" xml:space="preserve">
+    <value>image is built without record command line</value>
```

Comment:
>is built [](http://example.com/codeflow?start=17&length=8)

'image was compiled without {record command line}'
is it possible to have specific argument here (e.g. -grecord-gcc-switches) for each type of binary?

Reply:
fixed the first part, for the second part a binary can have many modules with different compiler so can not single one, will keep it as it is